docker/Dockerfile

```diff
@@ -14,7 +18,12 @@ RUN apt-get update \
         systemd \
         ca-certificates \
         adduser \
+    # Install Logging Agent.
     && curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | REPO_SUFFIX="$REPO_SUFFIX" DO_NOT_INSTALL_CATCH_ALL_CONFIG=true bash \
+    # Store versions in the VERSION file.
+    && echo REPO_SUFFIX=$REPO_SUFFIX >> /VERSION \
+    && dpkg -s google-fluentd | grep Version | sed 's/Version: /GOOGLE_FLUENTD_VERSION=/g' >> /VERSION \
+    && /opt/google-fluentd/embedded/bin/gem list fluent-plugin-google-cloud | tail -n 1 | sed 's/fluent-plugin-google-cloud (/FLUENT_PLUGIN_GOOGLE_CLOUD_VERSION=/g;s/)//g' >> /VERSION \
```

Comment:
Simpler: `sed 's/^fluent-plugin-google-cloud (\([^,]\+\)[^)]*)$/FLUENT_PLUGIN_GOOGLE_CLOUD_VERSION=\1/' >> /VERSION`

Reply:
I like that idea. What is the `(\([^,]\+\)[^)]*)` part trying to match? Seems it's trying to match:
a `(` followed by more than one character that is not `,` followed by a `)` followed by any number of characters that is not `)`.

The original line is in the format of `fluent-plugin-google-cloud (0.7.2)`.

This regex below seems to work. It might not be very strict, but probably sufficient in this case. WDYT?
```
/opt/google-fluentd/embedded/bin/gem list fluent-plugin-google-cloud | sed 's/^fluent-plugin-google-cloud (\(.*\))$/FLUENT_PLUGIN_GOOGLE_CLOUD_VERSION=\1/'
```

Reply:
Yeah, you're right that what you have in the new commit matches your original substitution. But didn't we have a case where, if multiple versions of a gem happened to be installed, they would appear as a comma-separated list in parentheses? That was the motivation for my more general regex.

Reply:
Oh, I see. That makes sense. The new regex should cover that. BTW we **should** never have two versions of `fluent-plugin-google-cloud` in the package.